### ISSUE TITLE
Incorrect detection of opportunistic security for IMAP

### ISSUE THREAD

**Issue by dilyanpalauzov** [drwetter/testssl.sh#1854]
I use v3.1 dev(3.0-472-g895a6b9).  
```
$ telnet mail.aegee.org 143
Trying 144.76.142.78...
Connected to mail.aegee.org.
Escape character is '^]'.
* OK [CAPABILITY IMAP4rev1 LITERAL+ ID ENABLE STARTTLS LOGINDISABLED] mail.aegee.org Cyrus IMAP 3.2.6-184-gf519597f7 server ready
l logout
* BYE LOGOUT received
l OK Completed
Connection closed by foreign host.
```
This means, that unless STARTTLS is utilized, login is disabled.

```
$ testssl -t imap mail.aegee.org:143
…
 Rating (experimental) 

 Rating specs (not complete)  SSL Labs's 'SSL Server Rating Guide' (version 2009q from 2020-01-30)
 Specification documentation  https://github.com/ssllabs/research/wiki/SSL-Server-Rating-Guide
 Protocol Support (weighted)  0 (0)
 Key Exchange     (weighted)  0 (0)
 Cipher Strength  (weighted)  0 (0)
 Final Score                  0
 Overall Grade                T
 Grade cap reasons            Grade capped to T. Encryption via STARTTLS is not mandatory (opportunistic).
```
Well, in this case STARTTLS is mandatory and testssl.sh has not detected this.

**Comment by drwetter** [comment]
Hi Diliyan,

there's another thread about this here.

TL;DR: enforcement of TLS will at some time be detected but still labeled as unsafe.

Cheers, Dirk

**Comment by drwetter** [comment]
See #1832, #1833
